.travis.yml

```diff
@@ -12,4 +12,4 @@ install:
   - gem install sass
   - gem install jekyll -v 3.6.2
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION test example/makeMicrosite
+  - sbt ++$TRAVIS_SCALA_VERSION scalafmtCheck test example/makeMicrosite
```

Comment:
btw, I don't think this does test configs or the sbt / project files.

Reply:
https://github.com/scalaz/scalaz/pull/1649